### DISCUSSION TITLE
[求助]新手绳匠如何快速提升绳网等级？

### DISCUSSION THREAD

**Discussion by Aris-Tendo** [share121/inter-knot#328] [其他]
![image](https://github.com/user-attachments/assets/c36b8d8d-7d65-46c1-ade0-ecf97baeccf8)

如题~刚当上绳匠没多久，在绳网上看到好多人和我一样都是1级，大家都是新手吗？但是，偶尔也会看到绳网等级超高的人，有什么快速升级的诀窍吗？

**Comment by cpu644** [comment]
多接委托 多发委托 促进新艾莉都的经济循环 :-)

**Comment by lappslover** [comment]
和绳网官方私通（
